browser/ui/views/tabs/brave_tab_strip.cc

```diff
@@ -428,6 +428,10 @@ void BraveTabStrip::UpdateTabContainer() {
     // In order to update shadow state, call ActiveStateChanged().
     tab_at(active_index.value())->ActiveStateChanged();
   }
+
+  // Called only at startup or vertical tab mode changed.
+  // To make initial tabs layout properly.
+  DeprecatedLayoutImmediately();
```

Comment:
:+1: - I think we still will need to get rid of this eventually but should be okay for now.